src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingVenueHelperTest.java

```diff
@@ -15,6 +15,9 @@
 import static uk.gov.hmcts.ecm.common.helpers.ESHelper.LISTING_VENUE_FIELD_NAME;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.ALL_VENUES;
 
+@SuppressWarnings({"PMD.UseProperClassLoader", "PMD.LawOfDemeter", "PMD.LinguisticNaming",
+    "PMD.ExcessiveMethodLength", "PMD.UnusedPrivateMethod", "PMD.JUnit5TestShouldBePackagePrivate",
```

Comment:
JUnit5TestShouldBePackagePrivate - is this required?

Reply:
Sorted